foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -162,7 +162,7 @@ Writing an HTML comment is simple: We just enclose the comment with `<!--` and `
 
  **VSCode keyboard shortcut** 
 
-If you find tying out the Comments syntax tiring, the following shortcut will help you quickly create a new comment, convert any line to a comment, or uncomment any line:
+If you find tiring out the comments syntax tiring, the following shortcut will help you quickly create a new comment, convert any line to a comment, or uncomment any line:
```

Comment:
I don't think this is what you meant to type 🤔 

>If you find tiring out the comments syntax tiring

I think this should be:

"If you find **typing** out the comments syntax tiring..."

Reply:
Also, would you still like the review from Kevin? I see you requested one but I think we're in the finishing up phase :)

Reply:
Good point there

Reply:
> I don't think this is what you meant to type 🤔 
> 
> 
> 
> >If you find tiring out the comments syntax tiring
> 
> 
> 
> I think this should be:
> 
> 
> 
> "If you find **typing** out the comments syntax tiring..."

Yes that's exactly what I meant. I'm tired so I'm exemplifying the statement lol 😂

Reply:
> I don't think this is what you meant to type 🤔 
> 
> 
> 
> >If you find tiring out the comments syntax tiring
> 
> 
> 
> I think this should be:
> 
> 
> 
> "If you find **typing** out the comments syntax tiring..."

Made the change